workbench/icons/Makefile.am

```diff
@@ -0,0 +1,20 @@
+icondir = $(datadir)/icons/hicolor/16x16/apps
```

Comment:
>> not sure what the dist_ adds here since the stuff in _DATA should be distributed anyway

> No, _DATA is not distributed by default, I guess because it's often generated out of templates and the like.

Makes sense, why would you want to distribute files that you've indicated need to be installed? &lt;/sarcasm&gt; 

In my experience it actually does distribute them, either that or it doesn't cause `make distcheck` to fail, or else I've always been very lucky and somehow got them distributed automatically.